lib/interactive.dart

```diff
@@ -7,33 +7,46 @@ import 'package:avf/firebase.dart' as fb;
 Logger logger = Logger("interactive.dart");
 
 const BS_ROW_CSS = "row";
+const BS_CONTAINER_CSS = "container";
```

Comment:
BS doesn't unpack nicely as an acronym for me - please could you expand

Reply:
Completely missed it :/ fixed